emm-docs/themes/hugo-geekdoc/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You can pretty much ignore everything under the `hugo-geekdoc` directory. It is a theme that was downloaded and extracted from [here](https://github.com/thegeeklab/hugo-geekdoc).